packages/api/drizzle/0003_posthog_analytics.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `config` ADD `enabled_analytics` integer DEFAULT true NOT NULL;--> statement-breakpoint
+ALTER TABLE `config` ADD `distinct_id` text DEFAULT 's0llcp0p79kcmf7hajobdg3lhc' NOT NULL;
```

Comment:
Is there a random id generator in sqlite to use here? I know this default should never actually be used but this will be confusing to read in the database and could lead to a future bug where everyone gets the same id.

Nit: I would prefer to name this `distinct_client_id`

Reply:
Yeah, I thought this name was not great. I can change to the one you suggest, it's not really used anywhere so the length isn't a problem.